Cargo.toml

```
@@ -15,8 +19,7 @@ members = [
 ]
 
 [dependencies]
-engula-storage = { package = "storage", path = "src/storage" }
-engula-journal = { package = "journal", path = "src/journal" }
-engula-microunit = { package = "microunit", path = "src/microunit" }
+microunit = { path = "src/microunit" }
+
 clap = "3.0.0-beta.5"
-tokio = { version = "1", features = ["full"] }
+tokio = { version = "1", features = ["full"] }
```

```suggestion
tokio = { version = "1", features = ["full"] }

```

I'll investigate whether we can add a editorconfig to ensure every file with a newline so that we don't see this kind of blank changes time after time.

---

Yes, I create an [issue](https://github.com/engula/engula/issues/120) for it.